docs/quantum_research/solver_parameters.rst

```diff
@@ -643,100 +640,14 @@ This example sets random linear and quadratic biases on all qubits.
-floating-point numbers identifying points in the schedule at which to change
-the gain applied to :ref:`parameter_qpu_h`. The first element in the pair is
-time, :math:`t` in microseconds |anneal_time_parameter_granularity|; the second,
-the unitless :math:`g` in the range :ref:`property_qpu_h_gain_schedule_range`.
-The resulting time-dependent gain is the piecewise-linear curve that connects
-the provided points over the same range of times as the
-:ref:`parameter_qpu_anneal_schedule`.
-
-The following rules apply to the set of gain points provided:
-
-*   Time :math:`t`, in microseconds, must increase for all points in the
-    schedule.
-*   The first point of time must be zero, :math:`t=0.0`.
-*   The last point of time must match the last time in the
-    :ref:`parameter_qpu_anneal_schedule` or the
-    :ref:`parameter_qpu_annealing_time`.
-*   The number of points must be :math:`\geq 2`.
-*   The steepest slope of any curve segment,
-    :math:`\frac{g_i - g_{i-1}}{t_i - t_{i-1}}`, must be within the bounds
-    supported by the selected QPU.\ [#]_  Note that the I/O system that delivers
-    the :ref:`parameter_qpu_h`-controlling waveform---the :math:`\Phi^x_i(s)`
-    term of equation :math:numref:`qpu_equation_rfsquid_hamiltonian` in the
-    :ref:`qpu_annealing` section---to a QPU limits bandwidth using a low-pass
-    filter with a cutoff frequency of 3 MHz for |dwave_5kq| systems and 30 MHz
-    for |adv2| systems; if you configure a too-rapidly changing curve, even
-    within the supported bounds, expect distorted values of
-    :ref:`parameter_qpu_h` for your problem.
-
-.. [#]
-    To see the supported slope for a particular QPU, submit a test problem with
-    slopes that are expected to violate any limitations; you can then read the
-    range of supported slopes in the returned error message. (Your account in
-    the Leap service is not charged for rejected problems.)
-
-    Supported slopes are typically under :math:`\frac{G_{max} - G_{min}}{0.02}`,
-    where :math:`G_{min}` and :math:`G_{max}` here stand for the range limits
-    of the time-dependent gain for the QPU. For example, for a QPU with
-    :ref:`property_qpu_h_gain_schedule_range` value of :code:`[-3, 3]`, a slope
-    above :math:`\frac{3 - (-3)}{0.02} = 300`, which occurs for a schedule that
-    contains :code:`[... [10.0, 0], [10.01, 3], ...]`, is likely to return an
-    error message with the maximum allowed slope.
-
-Default :math:`g(t)`, when left unspecified, is 1, which can be explicitly coded
-as
-
-.. code-block:: py
-
-    h_gain_schedule=[[0,1],[t_final,1]]
-
-where `t_final` is the requested annealing time.
-
+Sets a :ref:`time-dependent h-gain <qpu_qa_h_gain>` for linear coefficients
+(:ref:`qubit biases <parameter_qpu_h>`) in the Hamiltonian.
+    
 Relevant Properties
 -------------------
 
 *   :ref:`property_qpu_h_gain_schedule_range` defines the range of the
     time-dependent gain values permitted for the solver.
-
```

Comment:
See above about the audience of researcher learning about the features and programmer needing to see restrictions on inputs